engine/src/main/java/org/terasology/rendering/opengl/ScreenGrabber.java

```diff
@@ -92,14 +98,16 @@ public void takeScreenshot() {
      * If no screenshot data is available an error is logged and the method returns doing nothing.
      */
     public void saveScreenshot() {
-        final ByteBuffer buffer = FINAL.getColorBufferRawData();
+        FBO sceneFinalFbo = displayResolutionDependentFBOs.get(new ResourceUrn("engine:sceneFinal"));
```

Comment:
Using the .get() method here is ok, but I'd move the ResourceUrn into a static final field.